app/javascript/controllers/date_controller.js

```diff
@@ -1,22 +1,6 @@
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-  static targets = ["preselected"]
-
-  connect() {
-    if (!this.isDisabled(this.preselectedTarget)) {
-      if (this.isDraft(this.preselectedTarget)) {
-        this.element.classList.add("bg-danger", "text-light")
```

Comment:
This logic moved into the helper `calendar_day_tag` method, so it gets rendered server-side now.